go/pkg/tree/tree.go

```diff
@@ -63,6 +63,10 @@ func shouldIgnore(inp string, t command.InputType, excl []*command.InputExclusio
 // recursively), and loads their contents into the provided map.
 func loadFiles(execRoot string, excl []*command.InputExclusion, path string, fs map[string]*fileNode, chunkSize int, cache filemetadata.Cache) error {
 	absPath := filepath.Clean(filepath.Join(execRoot, path))
+	normPath, err := filepath.Rel(execRoot, absPath)
+	if err != nil {
```

Comment:
I don't think this actually ensures that absPath is under execRoot. filepath.Rel can return a path with ../

https://play.golang.com/p/V497MOsGPn_e